homeassistant/components/drop_connect/sensor.py

```diff
@@ -0,0 +1,272 @@
+
+
+SENSORS: dict[str, DROPSensorEntityDescription] = {
+    "current_flow_rate": DROPSensorEntityDescription(
+        key="current_flow_rate",
+        icon="mdi:shower-head",
+        native_unit_of_measurement="gpm",
+        suggested_display_precision=1,
+        value_fn=lambda device: device.current_flow_rate,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    "peak_flow_rate": DROPSensorEntityDescription(
+        key="peak_flow_rate",
+        icon="mdi:shower-head",
+        native_unit_of_measurement="gpm",
+        suggested_display_precision=1,
+        value_fn=lambda device: device.peak_flow_rate,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    "water_used_today": DROPSensorEntityDescription(
+        key="water_used_today",
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=1,
+        value_fn=lambda device: device.water_used_today,
+    ),
+    "average_water_used": DROPSensorEntityDescription(
+        key="average_water_used",
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.average_water_used,
+    ),
+    "capacity_remaining": DROPSensorEntityDescription(
+        key="capacity_remaining",
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.capacity_remaining,
+    ),
+    "current_system_pressure": DROPSensorEntityDescription(
+        key="current_system_pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement="psi",
+        suggested_display_precision=1,
+        value_fn=lambda device: device.current_system_pressure,
+    ),
+    "high_system_pressure": DROPSensorEntityDescription(
+        key="high_system_pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement="psi",
+        suggested_display_precision=0,
+        value_fn=lambda device: device.high_system_pressure,
+    ),
+    "low_system_pressure": DROPSensorEntityDescription(
+        key="low_system_pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement="psi",
+        suggested_display_precision=0,
+        value_fn=lambda device: device.low_system_pressure,
+    ),
+    "battery": DROPSensorEntityDescription(
+        key="battery",
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement="%",
+        suggested_display_precision=0,
+        value_fn=lambda device: device.battery,
+    ),
+    "temperature": DROPSensorEntityDescription(
+        key="temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement="°F",
```

Comment:
Wasn't this the celsius sensor?

Reply:
I changed the (unreleased) API in the DROP hub firmware to deliver °F to match our apps.  If I was going to use one value only, I might as well match our predominant market.

Reply:
Ah okay!